doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Block Size                                              | The number of bytes of a Block.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| C4                                                      | ClusterStor Colibri Control Canter, backend management server plus BUI based on Trinity.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Remove this, C4, please. It is outdated.

Reply:
Updated.